e2e/e2etests/e2etests.go

```diff
@@ -1478,12 +1479,19 @@ var AllE2ETests = []runner.E2ETest{
 		runner.WithMinimumVersion("v33.0.0"),
 	),
 	runner.NewE2ETest(
-
 		TestOpcodesName,
 		"test opcodes support in ZEVM",
 		[]runner.ArgDefinition{},
 		TestOpcodes,
 	),
+	runner.NewE2ETest(
+		TestRPCName,
+		"test json rpc support in ZEVM",
+		[]runner.ArgDefinition{
+			{Description: "tx hashes", DefaultValue: ""},
```

Comment:
Could we give more details about this argument? And eventually the test description as well, as it is unclear the association between `test json rpc support in ZEVM` and `tx hashes`

Reply:
i wanted to have a way to just provide some info from live networks or even locally with zetae2e run to check transactions directly

eg, we could have on nightly run one tx from before upgrade, one tx after upgrade on testnet, and this test would work